.github/workflows/ci.yml

```diff
@@ -24,3 +24,35 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+  visual:
```

Comment:
These still have the problem where they're flaky locally depending on OS font families?

Reply:
Yup. I didn't spend anytime looking at that this round.